### ISSUE TITLE
bot-community.py: ipfs-local flag does not work

### ISSUE THREAD

**Issue by clangenb** [encointer/encointer-node#132]
```
./bot-community.py init --ipfs-local                                                                                                                      1 err | 17:20:59 
didn't find ENCOINTER_CLIENT in env variables, setting client to ../target/release/encointer-client-notee
didn't find ENCOINTER_CLIENT in env variables, setting client to ../target/release/encointer-client-notee
IPFS environment not set up for using gateway
connecting to local chain
./my_keystore already contains 10 accounts.
Do you want to purge the ./my_keystore? [y, n]y
Purged the ./my_keystore.
 24.00 KiB / 24.00 KiB [=======================================================================================================================================================================================================] 100.00%
['QmQVQZSnWhZvgcMTtbX8pcssfATkgv9JQ4csZyKwEV8SZb']
initializing community
created bootstrappers: 5CAtcz7wiwTLCLKdXorfztUttKyJ2DMHCSMwJk9fhX7vo3mB 5FH9MFwJoHNja7B9qdQr7vU9cbzb9vzGUPH9Ho52Jn2WdCeP 5HjMD9ig1ZuQvvQjwK2nMY9VXVUq6AeRze2dRqT9NHGpFDZ5 5Gb49fRyrWVgC8PxHRZebjVjSun8sgKFD1iArbitgeGkkdnH 5DhvpVaaFAnEW
yHVPEAEQNnnYDczGV9FdxQT1w9FVEBUZfRN 5FRfTZZAYmEnh95EcnunCCynbjiPgaUEugeUGhmv54JKuJkh 5EUnCVeAU8pfZ9C4f9gz3P116ZR7Teifz3Qpf8YWxVAf6Dng 5CeqX97XryNLW9y6QeFmS3N176htaLCCY7yg32agmBYFib1M 5EevM5Si9tuWKSMCadERXckNSvVsh63udvcWxp1vRgmd7Sm9
5DcSE5nWaJ4SDPnXdUe1dATna9JWjUYg4MJo4VjxHb9uvLbX
Traceback (most recent call last):
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/connection.py", line 159, in _new_conn
    conn = connection.create_connection(
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/util/connection.py", line 80, in create_connection
    raise err
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/util/connection.py", line 70, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    httplib_response = self._make_request(conn, method, url,
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 355, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1255, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1250, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1010, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 950, in send
    self.connect()
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/connection.py", line 183, in connect
    conn = self._new_conn()
  File "/home/clang/.local/lib/python3.8/site-packages/urllib3/connection.py", line 168, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f63f1a60430>: Failed to establish a new connection: [Errno 111] Connection refused

```

I do run the daemon. Is something else needed?

```
ipfs daemon &                                                                                        ok | 15:55:59
[1] 26458
 ~  Initializing daemon...                                                                           ok | % | 15:56:03
go-ipfs version: 0.9.0
Repo version: 11
System version: amd64/linux
Golang version: go1.16.5
2021/11/22 15:56:03 failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/lucas-clemente/quic-go/wiki/UDP-Receive-Buffer-Size for details.
Swarm listening on /ip4/127.0.0.1/tcp/4001
Swarm listening on /ip4/127.0.0.1/udp/4001/quic
Swarm listening on /ip4/172.28.115.74/tcp/4001
Swarm listening on /ip4/172.28.115.74/udp/4001/quic
Swarm listening on /ip6/::1/tcp/4001
Swarm listening on /ip6/::1/udp/4001/quic
Swarm listening on /p2p-circuit
Swarm announcing /ip4/127.0.0.1/tcp/4001
Swarm announcing /ip4/127.0.0.1/udp/4001/quic
Swarm announcing /ip4/172.28.115.74/tcp/4001
Swarm announcing /ip4/172.28.115.74/udp/4001/quic
Swarm announcing /ip6/::1/tcp/4001
Swarm announcing /ip6/::1/udp/4001/quic
API server listening on /ip4/127.0.0.1/tcp/5001
WebUI: http://127.0.0.1:5001/webui
Gateway (readonly) server listening on /ip4/127.0.0.1/tcp/8080
Daemon is ready
```

**Comment by anizeani** [comment]
first, you don't need to run the local ipfs daemon. 
second: you don't have an ipfs problem. you just need to run the faucet.py script. The error is misleading, but see in your error log: 
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection ... 
you can't connect to the faucet which should be running on localhost:5000/api
more: for remote ipfs you should define env variables (see error messages in first lines of your log)
And in general: you can find all the info about it in the encointer book: https://book.encointer.org/developers-bot-communities.html
REMARK: I think on 0.6.0 it doesn't work anymore (again, faucet issue if I remember correctly). but I guess we can start looking forward and not try to fix backwards, right? (works with chain 0.7.0)

**Comment by clangenb** [comment]
Yes, this was indeed not an IPFS issue.
